rest_api.py

```diff
@@ -1,11 +1,13 @@
 """REST API."""
-
-import json
-from file_adapter import FileAdapter
-from filter import filter_releases
-from filter import sort
-from flask_restful import reqparse
-from flask_restful import Resource
+import sys
+sys.path.append('../')
+import json   # pylint: disable=g-import-not-at-top, g-bad-import-order
+from file_adapter import FileAdapter   # pylint: disable=g-import-not-at-top
+from filter import filter_releases   # pylint: disable=g-import-not-at-top
```

Comment:
filter seems to be a keywork in python, you might want to rename the modules to filters.